src/main/java/com/epam/test_generator/dto/PropertyDifferenceDTO.java

```diff
@@ -3,14 +3,14 @@
 public class PropertyDifferenceDTO {
 
     private String propertyName;
-    private String oldValue;
-    private String newValue;
+    private Object oldValue;
+    private Object newValue;
 
     public PropertyDifferenceDTO() {
 
     }
 
-    public PropertyDifferenceDTO(String propertyName, String oldValue, String newValue) {
+    public PropertyDifferenceDTO(String propertyName, Object oldValue, Object newValue) {
```

Comment:
Don't use Object. There are a lot of useful patterns to store different types. For example Either.